tideways/provision.sh

```diff
@@ -68,11 +71,19 @@ function restart_php() {
 
 function install_xhgui_frontend() {
     cp -f "${DIR}/nginx.conf" "/etc/nginx/custom-utilities/xhgui.conf"
+    declare -a packages=()
+    for version in "7.0" "7.1" "7.2" "7.3" "7.4"; do
+        if [[ $(command -v php$version) ]]; then
+            packages+=("php${version}-sqlite3")
+        fi
+    done
+    apt-get -y --allow-downgrades --allow-remove-essential --allow-change-held-packages -o Dpkg::Options::=--force-confdef -o Dpkg::Options::=--force-confnew install --fix-missing --fix-broken "${packages[@]}"
     if [[ ! -d "/srv/www/default/xhgui" ]]; then
         echo -e " * Git cloning xhgui from https://github.com/perftools/xhgui.git"
         cd /srv/www/default
-        noroot git clone "https://github.com/perftools/xhgui.git" xhgui
+        noroot git clone -b 0.16.0 "https://github.com/perftools/xhgui.git" xhgui
         cd xhgui
+        noroot composer config platform-check false
         echo " * Installing xhgui"
         sudo php install.php
```

Comment:
does this need to use `sudo`? That script downloads and runs composer then sets up cache folder permissions, wouldn't it cause issues being ran as `root`? `noroot` sees more appropriate